aiida/orm/importexport.py

```diff
@@ -583,6 +588,7 @@ def import_data_dj(in_path, user_group=None, ignore_unknown_nodes=False,
     detect the compression format (zip, tar.gz, tar.bz2, ...) and calls the
     correct function.
     :param in_path: the path to a file or folder that can be imported in AiiDA
+    :param user_group: Group wherein all imported Nodes will be placed.
```

Comment:
Here it is called `user_group` but in the generic `import_data` it is just `group`. I think the latter is better so let's keep that

Reply:
done